scripts/ci/install.sh

```diff
@@ -52,7 +52,7 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
 else
   if [ "$ARGV_OPERATING_SYSTEM" == "darwin" ]; then
     ./scripts/build/check-dependency.sh brew
-    brew install ccache jq --force-bottle
+    brew install ccache jq python --force-bottle
```

Comment:
Does installing python definitely also install pip? IIRC only python3 includes pip installed by default.

Reply:
Yup, it does intall pip, but might be that it ends up being `pip2` actually, and we might need to update the $PATH, now that I think about it.

Reply:
Let's see how the build comes back

Reply:
It's a good thing this is in `install.sh` and not `.travis.yml` ;-)